code/modules/mob/living/living_defines.dm

```diff
@@ -50,6 +50,8 @@
 
 	var/list/say_log = list() //a log of what we've said, plain text, no spans or junk, essentially just each individual "message"
 
+	var/bubble_icon = "default" //what icon the mob uses for speechbubbles

```

Comment:
Fixes include new vars and sometimes entire new systems.
By your logic they are features...